draft-ietf-rats-corim.md

```diff
@@ -1121,18 +1122,19 @@ The Reference Values Triple has the following structure:
 
 The `reference-triple-record` has the following parameters:
 
-* `ref-env`: Search criterion that locates an Evidence environment that matches the reference environment.
-* `ref-claims`: Search criteria that locates the Evidence measurements that match the reference Claims.
+* `ref-env`: Reference Environment Identity of the Target Environment
 
-To process `reference-triple-record` both the `ref-env` and `ref-claims` criteria are compared with Evidence entries.
+* `ref-claims`: One or more measurement claims for the Target Environment
+
+To process `reference-triple-record` both the `ref-env` and `ref-claims` criteria are compared with Evidence entries. First ref-env is used as a Search criterion to locate the Evidence environment that matches the reference environment. Subsequently, the ref-claims from this triple are used to match against the Evidence measurements for the matched environment.
```

Comment:
```suggestion
To process `reference-triple-record` both the `ref-env` and `ref-claims` criteria are compared with Evidence entries. First `ref-env` is used as a Search criterion to locate the Evidence environment that matches the reference environment. Subsequently, the ref-claims from this triple are used to match against the Evidence measurements for the matched environment.
```

Reply:
`ref-claims` in last sentence is also missing back quotes.
Note: It's easier to manage if each sentence starts a new line.

Reply:
Agree I will commit this suggestion